src/pages/case-study/learnerbly-accelerating-professional-growth.js

```diff
@@ -90,12 +81,15 @@ const IndexPage = props => {
       {/* "What was the problem we were tackling?" */}
       {shouldRender(data3) && (
         <Grid flex>
-          <BlockRow mobile={{ bottom: '4', top: '4' }} tablet={{ top: '6' }}>
+          <BlockRow
+            mobile={{ bottom: '4', top: '4' }}
+            tablet={{ top: '6' }}
+            style={{ justifyContent: 'space-between' }}
+          >
             <TextColumnsBlock
               data={normalise(data3)}
               colWidthOne={[1, 1, 1, 1, 5 / 12]}
               colWidthTwo={[1, 1, 1, 1, 6 / 12]}
-              middleColWidth={[0, 0, 0, 0, 1 / 12]}
```

Comment:
I missed this during the doctor link pr but if we want to space between columns that don't add up to 12 we can just use `space-between` to achieve this. Remember in general we want to keep the DOM as small as possible.

Reply:
oh yes! Indeed! I didn't think about that either. Great call 👍